Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,41 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
```

Comment:
The code in the two commands with this PR is nearly identical. I almost wonder if they should be refactored into a single command under the hood. But either way, it would be a shame to have them diverge, I think.